lib/urlwatch/reporters.py

```diff
@@ -614,3 +624,110 @@ def submit_to_slack(self, webhook_url, text):
 
     def chunkstring(self, string, length):
         return (string[0 + i:length + i] for i in range(0, len(string), length))
+
+
+class MarkdownReporter(ReporterBase):
```

Comment:
Can the `MarkdownReporter` share some/more code with the `TextReporter`? Seems like there's some code duplication there now.